bot/exts/help_channels/_cog.py

```diff
@@ -108,8 +109,10 @@ async def rename_help_post(self, ctx: commands.Context, *, title: str) -> None:
             # Silently fail in channels other than help posts
             return
 
-        if not await commands.has_any_role(constants.Roles.helpers).predicate(ctx):
-            # Silently fail for non-helpers
+        user_is_op = ctx.author.id == ctx.channel.owner_id
+        user_is_staff = await has_any_role_check(ctx, constants.Roles.helpers)
+
+        if not (user_is_op or user_is_staff):
             return
 
         await ctx.channel.edit(name=title)
```

Comment:
```suggestion
        if user_is_op or user_is_staff:
            await ctx.channel.edit(name=title)
```
I think not using a negative here makes the intent clearer